openstack/ims/v2/tags/ListImagesTags.go

```diff
@@ -0,0 +1,19 @@
+package tags
```

Comment:
file duplicated

Reply:
`ListImageTags` and `ListImagesTags` is diff.